### ISSUE TITLE
Redundant argument in sprintf

### ISSUE THREAD

**Issue by distahl** [lausser/check_nwc_health#229]
Hi,  
I'm getting this error, when using mode `count-connections`:  
  
```
Redundant argument in sprintf at ./check_nwc_health line 62826.
Redundant argument in sprintf at ./check_nwc_health line 62823.
WARNING - 537715 connections currently in use | 'connections'=537715;500000;750000;;
```
  
Additional Info:  
`check_nwc_health $Revision: 7.10.1.2 $`

The lines causing this are:  
```perl
    $self->add_info(sprintf '%d connections currently in use',
        $self->{cfwConnectionStatValue}||$self->{cfwConnectionStatCount}, $self->{usage});
```
and:  
```perl
    $self->add_info(sprintf '%d connections currently in use',
        $self->{cfwConnectionStatValue}, $self->{usage});
```
  
  
And I guess the `, $self->{usage}` part has to be removed, to make this warning disappear.
Would be great if you could fix this. Otherwise I'm forced to patch your script on every update myself again.

Thanks.

**Comment by lausser** [comment]
Thanks! Will be fixed in 7.10.1.7
